spec/client_spec.rb

```diff
@@ -109,6 +115,51 @@ def tokens
     end
   end
 
+  describe '#refund_invoice' do
+    subject { bitpay_client }
+    before { stub_const('ENV', {'BITPAY_PEM' => PEM}) }
+    it { is_expected.to respond_to(:refund_invoice) }
+    
+    it 'should get the token for the invoice' do
+      bitpay_client.refund_invoice(id: 'TEST_INVOICE_ID')
```

Comment:
While this looks like a constant it is really just the string value from the fixture and could really be any value - this pattern is consistent through all the client_spec tests.